### ISSUE TITLE
Text override for symbols

### ISSUE THREAD

**Issue by steadynow** [mamuso/sketch-truncate-textlines#1]
Thanks for this great & much needed feature.

Curious is it possible to extend the functionality so if apply a truncate rule to text in a symbol that it works for text overrides?

**Comment by MaeveG** [comment]
> Curious is it possible to extend the functionality so if apply a truncate rule to text in a symbol that it works for text overrides?

Would really love to see this, it's the main reason I'm looking for a plugin like this.

**Comment by mamuso** [comment]
Reporting progress: I gave this a try, and when I resize the text field, it breaks other symbol instances. I'm going to try a different approach and report back

**Comment by alexeiboronnikov** [comment]
Any progress on that? Would be handy. Thanks!

**Comment by diananoias** [comment]
Yes, please, I love the simplicity of this plugin. It would be actually so useful if it propagates the behavior of the symbol to all its instances! Thanks for the work done until now 🤗
